pyne/dbgen/simple_xs.py

```diff
@@ -56,10 +56,10 @@ def grab_kaeri_simple_xs(build_dir=""):
     for element in nucname.name_zz.keys():
         htmlfile = element.upper() + ".html"
         if htmlfile not in already_grabbed:
-            grab_kaeri_nuclide(element.upper(), build_dir)
-        all_nuclides = all_nuclides | parse_for_all_isotopes(
-            os.path.join(build_dir, htmlfile)
-        )
+            pass
+        else:
+            all_nuclides = all_nuclides | parse_for_all_isotopes(os.path.join(build_dir, 
+                                                                          htmlfile))
```

Comment:
I think we can remove not from the if condition, thus we can also remove pass and else conditions.
```suggestion
        if htmlfile in already_grabbed:
            all_nuclides = all_nuclides | parse_for_all_isotopes(os.path.join(build_dir, 
                                                                          htmlfile))
```